docs/main/README.md

```diff
@@ -178,6 +178,13 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
     oc apply -n orchestrator -f https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/refs/heads/main/config/samples/_v1alpha1_orchestrator.yaml
     ```
 
+### Using Knative kafka broker
```

Comment:
For consistency, let's use `Kafka` (initcap)

Reply:
s/kafka/Kafka

Reply:
but but but how, I used search&replace for that string for everywhere in the project, why this  one was missed ??!!

Reply:
it was the first one and for that, it was ignored :-)